server/proto/management.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package management;
+
+enum Tee {
+    Sgx = 0;
+    Tdx = 1;
+    SevSnp = 2;
+    Sample = 3;
+}
+
+message GetPolicyRequest {
+    Tee tee = 1;
+    optional string user = 2;
+}
+message GetPolicyResponse {
+    bytes content = 1;
+}
+
+message GetReferenceDataRequest {
+    Tee tee = 1;
+    optional string user = 2;
+}
+message GetReferenceDataResponse {
+    bytes content = 1;
+}
+
+message SetPolicyRequest {
+    Tee tee = 1;
+    optional string user = 2;
+    bytes content = 3;
+}
+message SetPolicyResponse {}
+
+message SetReferenceDataRequest {
+    Tee tee = 1;
+    optional string user = 2;
+    bytes content = 3;
+}
+message SetReferenceDataResponse {}
+
+message RestoreDefaultPolicyRequest {
+    Tee tee = 1;
+    optional string user = 2;
+}
+message RestoreDefaultPolicyResponse {}
+
+message RestoreDefaultReferenceDataRequest {
+    Tee tee = 1;
+    optional string user = 2;
+}
+message RestoreDefaultReferenceDataResponse {}
+
+message TestOpaRequest {
+    bytes policy = 1;
+    bytes reference = 2;
+    bytes input = 3;
+}
+message TestOpaResponse {
+    bytes status = 1;
+}
+
+service ManagementService {
+    rpc GetPolicy(GetPolicyRequest) returns (GetPolicyResponse) {};
+    rpc GetReferenceData(GetReferenceDataRequest) returns (GetReferenceDataResponse) {};
+    rpc SetPolicy(SetPolicyRequest) returns (SetPolicyResponse) {};
+    rpc SetReferenceData(SetReferenceDataRequest) returns (SetReferenceDataResponse) {};
+    rpc RestoreDefaultPolicy(RestoreDefaultPolicyRequest) returns (RestoreDefaultPolicyResponse) {};
+    rpc RestoreDefaultReferenceData(RestoreDefaultReferenceDataRequest) returns (RestoreDefaultReferenceDataResponse) {};
+    rpc TestOpa(TestOpaRequest) returns (TestOpaResponse) {};
```

Comment:
That one definitely needs a comment here. What is this for? I find it suspicious that we have a test endpoint as part of our main API. Maybe we want to have a separate test service, that could be optionally implemented.

Reply:
OK, I will remove this testing API.